packages/block-editor/src/components/list-view/appender.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId } from '@wordpress/compose';
+import { speak } from '@wordpress/a11y';
+import { useSelect } from '@wordpress/data';
+import { forwardRef, useState, useEffect } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+import useBlockDisplayTitle from '../block-title/use-block-display-title';
+import Inserter from '../inserter';
+
+export const Appender = forwardRef(
+	( { nestingLevel, blockCount, clientId, ...props }, ref ) => {
+		const [ insertedBlock, setInsertedBlock ] = useState( null );
+
+		const instanceId = useInstanceId( Appender );
+		const { hideInserter } = useSelect(
+			( select ) => {
+				const { getTemplateLock, __unstableGetEditorMode } =
+					select( blockEditorStore );
+
+				return {
+					hideInserter:
+						!! getTemplateLock( clientId ) ||
+						__unstableGetEditorMode() === 'zoom-out',
```

Comment:
What's the need for `__unstableGetEditorMode() === 'zoom-out'`?

It stands out, because zoom out mode is something that applies to the editor canvas, and List View is separate to the editor canvas, so I'm wondering how the two might be linked.

It looks like it isn't possible to access List view in zoomed out mode (though it does stay open when already open, which seems like a bug).

Reply:
Looks to me like it's copy/pasted from packages/block-editor/src/components/block-list-appender/index.js. Maybe there's some benefit to keeping these files in sync where possible even though this line isn't needed, with the hope that we can share more code in the future?

Reply:
I would probably lean towards removing it, but it's not critical and definitely not a blocker.